zerver/lib/test_classes.py

```diff
@@ -1258,7 +1258,9 @@ def check_has_permission_policies(
         self.assertFalse(validation_func(guest_user))
 
     @contextmanager
-    def tornado_redirected_to_list(self, lst: List[Mapping[str, Any]]) -> Iterator[None]:
+    def tornado_redirected_to_list(
+        self, lst: List[Mapping[str, Any]], expected_num_events: int = 1
```

Comment:
What do you think about just making `expected_num_events` a mandatory kwarg, e.g. 
```
def tornado_redirected_to_list(
        self, lst: List[Mapping[str, Any]], * expected_num_events: int
)
```

We're effectively doing that when the number isn't 1 already, and it might be nice to just have number explicit in every line using this, not just when the number is not 1.